style.css

```diff
@@ -529,3 +588,605 @@ textarea::placeholder {
+    align-items: center;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    gap: 12px;
+    width: 1156px;
+    height: 496px;
+  }
+
+  .portfolio-desktop1 {
+    display: inline-block;
+  }
+
+  .portfolio-card1 {
+    display: none;
+  }
+
+  .project-desc {
+    width: 544px;
+    height: 445px;
+    left: 590px;
+    padding: 24px;
+  }
+
+  .project-title {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    width: 496px;
+    height: 46px;
+    margin-top: 2px;
+    margin-bottom: 12px;
+  }
+
+  .frame2 {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 32px;
+    justify-content: flex-start;
+    align-content: center;
+  }
+
+  .frame2 > img {
+    margin: 0 12px;
+  }
+
+  .frame2 > h4 {
+    color: #7a869a;
+  }
+
+  .text {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .tag {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    width: 203px;
+    height: 24px;
+  }
+
+  ul.tag {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    border-radius: 8px;
+    gap: 8px;
+    list-style: none;
+  }
+
+  button#skill {
+    color: #6070ff;
+    cursor: pointer;
+    background: #ebebff;
+    border-radius: 8px;
+    font-family: 'Poppins', sans-serif !important;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    border: 0;
+    height: 24px;
+    padding: 4px 12px;
+  }
+
+  button#skill:enabled {
+    background: #ebebff;
+    color: #7f8cff;
+  }
+
+  button#skill:hover {
+    background: #6070ff;
+    color: #fff;
+  }
+
+  button#skill:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  button#skill:disabled {
+    background: #fff;
+    color: #5e6c84;
+  }
+
+  .normal-button {
+    font-family: 'Poppins', sans-serif !important;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    border: 1px solid #6070ff;
+    cursor: pointer;
+    border-radius: 8px;
+    padding: 12px;
+    background: #fff;
+    width: 124px;
+    margin-top: 40px;
+  }
+
+  .normal-button:enabled {
+    background: #fff;
+    color: #7f8cff;
+  }
+
+  .normal-button:hover {
+    background: #6070ff;
+    color: #fff;
+  }
+
+  .normal-button:active {
+    background: #2230d2;
+    color: #fff;
+  }
+
+  .normal-button:disabled {
+    background: #fff;
+    color: #5e6c84;
+  }
+
+  .about-intro {
+    width: 100%;
+    padding: 166px 24px;
+  }
+
```

Comment:
- [ ] The `padding-top` of the _**About**_ section is too much, I would advise you to reduce it so that your design looks identical to that of the Figma template that you chose. 👍 

### Your Design

![Screenshot4](https://user-images.githubusercontent.com/77106962/222579308-5cedec49-5b3f-44a5-b162-ff88e1e8eeeb.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/222579374-340f5b85-b3ef-4751-a33f-93a788c02602.png)

Reply:
@Gambit142 I removed the padding-top width